### PR TITLE
Convert order completions to new list utilities

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/list_completions.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+          "iSortPriority" : 0,
+          "include" : true
+        
+        }, {
+          "sTitle" : "Completed",
+          "mData" : "completed",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/list_completions.js]: :100: to `nonZero`

File: miso-web/src/main/webapp/scripts/list_completions.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        }, {
+          "sTitle" : "Remaining",
+          "mData" : "remaining",
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Last Updated",
+          "mData" : "lastUpdated",
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/list_completions.js]: This is pretty verbose (displays down to the millisecond). Would you mind adjusting this in the DTO?

Comment (apmasell) [inline miso-web/src/main/webapp/scripts/list_completions.js]: It's just date now. Interestingly, this is the date time format we have everywhere (that's a zone not millis), I'm just not sure it ever gets displayed..

Comment (callunity) [inline miso-web/src/main/webapp/scripts/list_completions.js]: Thanks for making the change!

**Pull request merged**